test/exit.tsx

```diff
@@ -54,7 +54,7 @@ test.serial('exit on unmount() with raw mode', async t => {
 
 test.serial('exit with thrown error', async t => {
 	const output = await run('exit-with-thrown-error');
-	t.true(output.includes('errored'));
```

Comment:
Currently test pass because we print error message in [ErrorOverview](https://github.com/vadimdemedes/ink/blob/master/src/components/ErrorOverview.tsx)

Reply:
Could you undo all changes to existing tests and create new tests instead? I want to make sure existing functionality doesn't break.

Reply:
done